operator/config/crd/patches/cainjection_in_operations.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    certmanager.k8s.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
can you please explain in detail how this boiler plate works and why is it needed?

Reply:
Well this is part of the kubebuilder generated code, and there is a lot going on there (see https://book.kubebuilder.io/reference/reference.html). If you want I will be happy to organize another walk through sharing what I have learned so far 

Please note that some of these staff will be removed later on when we will start working on controllers, but this is out of the scope of this PR